srcpkgs/parpd/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'parpd'
+pkgname=parpd
+version=1.7
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+make_install_args="BINDIR=/usr/bin"
+short_desc="RFC 1027 compliant Proxy ARP Daemon"
+maintainer="Arsen Arsenović <arsen@aarsen.me>"
+license="BSD-2-Clause"
+homepage="https://roy.marples.name/projects/parpd/"
+distfiles="https://roy.marples.name/downloads/parpd/parpd-${version}.tar.xz"
+checksum=f1f2ed753243b86205922ab0ec1cd0f426779e6b8c273206f0568e819a3fe3b4
+conf_files=/etc/parpd.conf
+
+post_install() {
+	vsv parpd
+	vlicense LICENSE
+
+	vconf "${FILESDIR}/parpd.conf"
```

Comment:
Why do we need to install the config file when it only contains a comment?

Reply:
The daemon won't start without it, and, IMO, it helps to have a blank/default configuration at least so that it's easily guessable by admins.